docs/source/visualisation/experiment_tracking.md

```diff
@@ -173,9 +188,25 @@ You can now access, compare and pin your runs by toggling the `Compare runs` but
 
 ![](../meta/images/experiment-tracking_demo.gif)
 
-## View and compare plot data
+## View and compare plots
+
+From Kedro-Viz `>=5.0.0` experiment tracking also supports the display and comparison of plots, such as Plotly and Matplotlib. This section will show you how to enable this functionality to leverage Matplotlib. [Similar instructions can be followed for Plotly](../visualisation/visualise_charts_with_plotly.md). 
+
+### Update the dependencies
 
-From Kedro-Viz version 5.0.0 experiment tracking also supports the display and comparison of plots, such as Plotly and Matplotlib.
+You must update the `src/requirements.txt` file in your Kedro project by adding the following dataset to enable Matplotlib for your project:
+
+```bash
+kedro[matplotlib.MatplotlibWriter]==0.18.3
```

Comment:
True! I have to make some wholesale changes to the other viz docs when that PR merges so can remove this then if you merge this PR ahead of #1214 but otherwise, if it won't merge in until later, you can certainly remove this.

Reply:
I've removed it since this PR will be merged in the next couple of days.